### PR TITLE
Alibaba Cloud, Hosteur, etc.

### PR DESCRIPTION
**Pull request by alaz**

### CODE REVIEW

**Comment by shawnps** [comment]
@alaz thanks for the PR. Would you mind removing the `ColoCrossing` entry? Looks like we already have it in there for a larger range:

```
➜  ipcat [master] go run cmd/ipcat/main.go -l 172.245.253.0
2018/02/20 10:55:39 Loaded 3228 entries
[172.245.0.0:172.245.255.255] ColoCrossing http://www.colocrossing.com/
```

**Comment by alaz** [comment]
@shawnps ah, missed it. Done.

File: datacenters.csv [hunk 1]
```diff
@@ -1282,6 +1286,7 @@
 89.47.16.0,89.47.31.255,everhost.ro,http://everhost.ro/
 89.106.0.0,89.106.31.255,Grid Telekom,http://grid.com.tr/
 89.107.16.0,89.107.23.255,Telecity Group,http://www.ukgrid.net/
+89.111.176.0,89.111.179.255,r01,https://r01.ru/
```
Comment (shawnps) [inline datacenters.csv]: @alaz just curious how did you find that this range is r01.ru? I don't see it mentioned in `whois`, and `host` lookups on some of the IPs in the range give me hostnames ending in `hc.ru`

Comment (alaz) [reply datacenters.csv]: From `whois`:

```
route:          89.111.176.0/21
descr:          JSC Centrohost route
origin:         AS41126
mnt-by:         RUNIC-MNT
created:        2015-02-25T15:41:27Z
last-modified:  2015-05-13T15:14:32Z
source:         RIPE
```

[ipinfo on AS41126](https://ipinfo.io/AS41126): related domain is `r01.ru`

<details>
<summary>`centrohost.ru` issues redirect to `r01.ru`</summary>

```
$ curl -s http://centrohost.ru
<html>
<meta http-equiv='cache-control: no-cache, must-revalidate'>
<meta http-equiv='pragma' content='no-cache'>
<meta http-equiv="refresh" content="0; URL=https://r01.ru">
```
</details>

Comment (shawnps) [reply datacenters.csv]: @alaz great, thanks!

**Comment by shawnps** [comment]
Thanks @alaz!

**Pull request merged**